liquibase-service/src/main/resources/db/rdb/routines/001-generate_data_for_data_sync_config-005.sql

```diff
@@ -0,0 +1,81 @@
+-- Removing obsoleted query
```

Comment:
@alog-enquizit Could you please check the changelog for the rdb and confirm if we need to add in the changes? What would the runtime expectation in demo be?

Reply:
@ndduc01 You might want to update the databasechange log to include the scripts for liquibase to pick it up and run them.

https://github.com/CDCgov/NEDSS-DataReporting/blob/main/liquibase-service/src/main/resources/db/changelog/db.rdb.changelog-16.1.yaml

You might have to do something like this:

```changeSet:
      id: 2
      author: liquibase
      changes:
        - sqlFile:
            path: 001-create_data_sync_config-002.sql
            splitStatements: false
```

Reply:
so I need to do it for 001-create_data_sync_config-003.sql, then 004 and so on as well?